public/navigation.json

```diff
@@ -8521,6 +8521,113 @@
             }
           ]
         },
+        {
```

Comment:
Mover todo o conteúdo entre 8524 e 8628 para fora do objeto Promotions & Taxes API, logo antes da Reviews and Ratings API, para que apareça como uma categoria própria entre essas duas APIs.